### PR TITLE
add aws-sam-cli, chevron, aws_lambda_builders, serverlessrepo

### PR DESCRIPTION
**Pull request by rluria14**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by rluria14** [comment]
@oblute @ndmaxar will you be maintainers?

File: recipes/aws-lambda-builders/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    - aws_lambda_builders.workflows.go_modules
+    - aws_lambda_builders.workflows.java_gradle
+    - aws_lambda_builders.workflows.java_maven
+    - aws_lambda_builders.workflows.nodejs_npm
+    - aws_lambda_builders.workflows.python_pip
+    - aws_lambda_builders.workflows.ruby_bundler
+
+about:
+  home: "https://github.com/awslabs/aws-lambda-builders"
+  license: Apache Software
```
Comment (marcelotrevisani) [inline recipes/aws-lambda-builders/meta.yaml:45]:
```suggestion
  license: Apache-2.0
```

File: recipes/aws-sam-cli/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    - samcli.local.lambdafn
+    - samcli.local.layers
+    - samcli.local.services
+    - samcli.settings
+  commands:
+    - sam --help
+
+about:
+  home: "https://github.com/awslabs/aws-sam-cli"
+  license: Apache Software
```
Comment (marcelotrevisani) [inline recipes/aws-sam-cli/meta.yaml:95]:
```suggestion
  license: Apache-2.0
```

File: recipes/serverlessrepo/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    - six >=1.11,<2
+
+test:
+  imports:
+    - serverlessrepo
+    - tests.unit
+
+about:
+  home: "https://github.com/awslabs/aws-serverlessrepo-python"
+  license: Apache Software
```
Comment (marcelotrevisani) [inline recipes/serverlessrepo/meta.yaml:34]:
```suggestion
  license: Apache-2.0
```

**Comment by marcelotrevisani** [comment]
If you want to, next time you can give a go to [grayskull :skull: ](https://github.com/marcelotrevisani/grayskull/) to generate recipes for conda looking for packages on pypi

**Comment by rluria14** [comment]
> If you want to, next time you can give a go to [grayskull 💀 ](https://github.com/marcelotrevisani/grayskull/) to generate recipes for conda looking for packages on pypi

sounds good! will give it a try next time

**Comment by rluria14** [comment]
@conda-forge/staged-recipes ready for review

**Pull request merged**